### ISSUE TITLE
[Bug] Challenge Trackers created by players are not shown to others.

### ISSUE THREAD

**Issue by SalieriC** [Larkinabout/fvtt-challenge-tracker#9]
Challenge trackers that are created using a macro by players seem to open up and disappear immediately for all players and GMs except the one who created the tracker. This is with the show property set to true. Here is the macro I was using:
```
const user = game.user
const borderColour = user.color
const segmentColour = "#05C3DD"
ChallengeTracker.open(12, 0, {show: true, title: `${user.name}'s Tokens`, frameColor: borderColour, outerColor: segmentColour,  windowed: false, persist: false})
```
(Also happens if windowed is set to true.)

**Comment by Larkinabout** [comment]
Hi @SalieriC, do you have the 'Show to Others' module setting set to the role of your players?:
![challenge-tracker-module-setting](https://user-images.githubusercontent.com/105953297/179933390-3dee1c9c-8b47-424f-acaf-22288742c3a0.png)

This should keep the challenge tracker open for everyone and allow the player that owns the challenge tracker to show and hide it as they wish.

It looks like there is an issue with challenge trackers appearing for other players momentarily where the player does not have permission. I'll fix that one in the next release.

**Comment by SalieriC** [comment]
Ah, it wasn't immediately obvious to me that this is tied to a setting. I had expected that at least I as the GM would see the players tracker. Thank you.
Due to the other part of your reply, I'll leave this open.

**Comment by Larkinabout** [comment]
It could perhaps do with an option for players to display the challenge tracker to the GM only for cases where a player and GM want it to be private, though I'll have to think about that one to avoid it getting messy.

I'll also update the README so it's clear that the `show: true` parameter only works alongside the module setting.

**Comment by Larkinabout** [comment]
Fixed in release 0.8.
